node_application_finder.go

```diff
@@ -0,0 +1,37 @@
+package nodestart
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+)
+
+type NodeApplicationFinder struct{}
+
+func NewNodeApplicationFinder() NodeApplicationFinder {
+	return NodeApplicationFinder{}
+}
+
+func (n NodeApplicationFinder) Find(workingDir string) (string, error) {
+
```

Comment:
So I am trying to think again about returning the absolute path here.
Does the spec give a guarantee that (1) the absolute path of workingDir during build, and (2) the absolute path of workingDir during launch would be equal?
If not, couldn't they be mounted a different points? If that's the case, we should possibly rely on relative paths.
Could you investigate?

Reply:
I'm asking for some clarification in CNB slack: https://buildpacks.slack.com/archives/CSFE49VGR/p1615320551022100

Reply:
Given the above, I agree it's probably a better practice to use a relative path.